CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+##  v1.10.0 (12/14/2020)
+
+**Common**
+
+* Added a shared_mutex class.
```

Comment:
* Ported the `std::shared_mutex` class from the C++17 Standard Library to be able to use it in C++11/14 code.